channels.yaml

```diff
@@ -1,8 +1,8 @@
 releases:
-- version: v1.17.5+k3s1
+- version: v1.17.6+k3s1
   minChannelServerVersion: v2.4.0-rc1
   maxChannelServerVersion: v2.4.99
-- version: v1.18.2+k3s1
+- version: v1.18.3+k3s1
   minChannelServerVersion: v2.4.4-rc1
```

Comment:
Please update the min Channel Server version to v2.4.5-rc1

Reply:
Updated. Cleaned commit history please review.